paymentintent.go

```diff
@@ -190,6 +190,7 @@ type PaymentIntentPaymentMethodOptionsCardInstallmentsParams struct {
 type PaymentIntentPaymentMethodOptionsCardParams struct {
 	Installments        *PaymentIntentPaymentMethodOptionsCardInstallmentsParams `form:"installments"`
 	MOTO                *bool                                                    `form:"moto"`
+	Network             *string                                                  `form:"network"`
```

Comment:
Would this normally be one of those type alias "enum"s eg
```
const (
 NetworkAmex
 NetworkCartesBancaires
 NetworkDiners
...
```
or do we not do that always?

Reply:
We don't use enums for params because those are pointers and you'd need a custom converter for every single enums.